enhancements/monitoring/hypershift-monitoring.md

```diff
@@ -0,0 +1,390 @@
+last-updated: 2021.12.08
+status: implementable
+---
+
+# HyperShift Metrics Monitoring
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Operational readiness criteria is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created.
+  - See [Documentation Required](#documentation-required) for further details.
+
+## Summary
+
+We propose to extract metric storage, alerting and querying to a separate service that can be scaled, deployed and managed in a decoupled mode. This will allow robust multi-cluster, fleet monitoring with best experience possible for cluster admins. This means SD org, running managed HyperShift will use RHOBS managed by Observability Group and AppSRE. 
+
+## Glossary
+
+* HSRE: HyperShift Admin SRE: People that will operate the Hypershift management cluster and its control planes.
+* CUS: Customers. People that create clusters that are managed by Hypershift.
+* BU: Business Unit. 
+* MS-SRE/MT-SRE: Managed Service / Managed Tenant Site Reliability Engineer.
+* RHOBS: Red Hat Observability Service.
+* [Observatorium](http://observatorium.io/): Project for scalable SaaS based monitoring and observability.
+
+For additional acronyms, see [Red Hat Dictionary](https://source.redhat.com/groups/public/red-hat-dictionary-or-lexicon/red_hat_abbreviations_acronyms_and_initialisms_dictionary_wiki)
+
+## Background
+
+[HyperShift project](https://github.com/openshift/hypershift) brings some changes to how OpenShift clusters can be deployed. The 10,000 feet view is to decouple the OCP control plane from the data plane. Control plane components (e.g. etcd, API servers, …) run on an underlay cluster, while worker components run remote on a different infrastructure.
+
+This is similar to how AWS, Google and so on run their managed Kubernetes service: You provide the worker nodes and they provide the control plane.
+
+The core idea of HyperShift topology is that it allows sharing infrastructure on control planes and allows the “hosted clusters” (workers) to have a lower computation and operational overhead.
+
+### Important HyperShift Assumptions
+
+* One-way communication. HyperShift management cluster can “talk to” the data plane, but not the opposite (except Kube API, Ignition and OIDC endpoint).
+* HyperShift is managed first, but form factor will be offered for self-management by 4.11.
+* There is tight budget on computation for control planes (outside of MVP).
+
+## Motivation
+
+This document enhancement proposes potential solution for Admin and Customer metric monitoring within those new topologies, since we cannot apply easily our current platform monitoring products using within cluster monitoring operator ([CMO](https://github.com/openshift/cluster-monitoring-operator)) easily (see [Alternatives](#use-pure-platform-monitoring-solution) on why).
```

Comment:
```suggestion
This document enhancement proposes a potential solution for Admin and Customer metric monitoring within those new topologies, since we cannot easily apply our current platform monitoring products using cluster monitoring operator ([CMO](https://github.com/openshift/cluster-monitoring-operator)) (see [Alternatives](#use-pure-platform-monitoring-solution) on why).
```